binary/script/embed.ipxe

```diff
@@ -25,7 +25,11 @@ echo Booting from net${idx}...
 autoboot net${idx}
 
 :autoboot
-autoboot
+set attempt:int32 0
+:retry
+iseq ${attempt} 5 && error-with-autoboot
```

Comment:
should this be `iseq ${attempt} 5 && goto error-with-autoboot`

Reply:
ah, yes! good catch. will update

Reply:
updated